package.json

```diff
@@ -63,7 +63,7 @@
     "realitio-gnosis-proxy": "github:gnosis/realitio-gnosis-proxy",
     "replace-in-file": "^6.1.0",
     "should": "^13.2.3",
-    "truffle": "^5.1.18",
+    "truffle": "5.1.35",
```

Comment:
Interesting, should Truffle be version pinned?

Reply:
nope, fixed to `^5.1.35`